### DISCUSSION TITLE
Does it support overriding the Files service?

### DISCUSSION THREAD

**Discussion by Tnze** [CodinGame/monaco-vscode-api#300] [Q&A] [answered]
Currently there is a memory filesystem to be add by default. Is there way to replace it with my own implementation?
Or can it be just disabled?

**Comment by cemalgnlts** [comment, marked as answer]
Here are examples on how to add a file system:

* https://github.com/CodinGame/monaco-vscode-api/blob/main/demo/src/features/filesystem.ts#L83

`RegisteredFileSystemProvider` class is also here: https://github.com/CodinGame/monaco-vscode-api/blob/main/src/service-override/files.ts#L107
